### ISSUE TITLE
Related to issue #39, placeholder never loads

### ISSUE THREAD

**Issue by abacaj** [mchome/flutter_advanced_networkimage#43]
Last commit: https://github.com/mchome/flutter_advanced_networkimage/pull/40 resolved this but was undone.

Given a bad image (where the file is corrupt): https://www.politico.com/apple-touch-icon-180x180.png will cause the image loader to throw an `operation failed` exception on `image codec`.

The code found in `flutter_advanced_networkimage.dart` throws an exception at ` ui.instantiateImageCodec(imageData)`:
```
    if (imageData != null) {
      print(imageData);
      if (key.loadedCallback != null) key.loadedCallback();
      return await ui.instantiateImageCodec(imageData);
    }
```

![image](https://user-images.githubusercontent.com/7272343/45037769-b74b9180-b02d-11e8-9e7d-b6b51dbd9125.png)

The response is `200` but the image is corrupt, i.e need to add a `try / catch` around the `ui.instantiateImageCodec(imageData);`  and load the placeholder.

**Comment by abacaj** [comment]
👍

**Comment by mchome** [comment]
Fixed in `0.3.9` :)
